updater/bin/fetch_files.rb

```diff
@@ -15,7 +15,7 @@ class UpdaterKilledError < StandardError; end
 trap("TERM") do
   puts "Received SIGTERM"
   error = UpdaterKilledError.new("Updater process killed with SIGTERM")
-  tags = { update_job_id: ENV.fetch("DEPENDABOT_JOB_ID", nil) }
+  tags = { "gh.dependabot_api.update_job.id": ENV.fetch("DEPENDABOT_JOB_ID", nil) }
```

Comment:
If we were logging the GitHub Repo ID we'd use `gh.repo.id` because it's owned by GitHub. Similarly since the UpdateJob's ID is owned by dependabot-api we should use `gh.dependabot_api.update_job.id` here.